library/include/rocwmma/internal/transforms_impl.hpp

```diff
@@ -181,9 +185,88 @@ namespace rocwmma
         lo          = Blend::Zip32::exec(lo, rot_hi);
```

Comment:
mark to replace with dpp::move

Reply:
Added TODO comment

Will do it in the next commit.